docs/src/routes/docs/components/dot.svx

```diff
@@ -0,0 +1,86 @@
+---
+name: Dot
+---
+
+<script>
+  import { Dot, Button } from 'attractions';
+  import Showcase from 'src/containers/docs/showcase.svelte';
+  import ColorPreview from 'src/components/docs/color-preview.svelte';
+</script>
+
+# Dot
+
+A dot.
+
+<Showcase>
+  <div slot="showcase" class="padded">
+    <Button>
+      Plain ol' dot: <Dot />
```

Comment:
I know that with this thing you meant to demonstrate that dots don't have margin by default, but it's such an eye-sore, so I'd rather you added margin there as well